<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mvd::ImportImagesDialog</class>
 <widget class="QWidget" name="mvd::ImportImagesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>115</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="MultiResolutionPyramidWidget" name="pyramidWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="filenamesListView"/>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="applyAllCheckBox">
     <property name="text">
      <string>???</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiResolutionPyramidWidget</class>
   <extends>QWidget</extends>
   <header>mvdMultiResolutionPyramidWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
